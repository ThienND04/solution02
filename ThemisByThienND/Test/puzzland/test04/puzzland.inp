32
4 4
K Y Q P 
K Q
Y P
Y Q
K P
4 5
U R E I 
U R
R E
U E
E I
R I
10 39
E V D R G S P Z J W 
S P
G W
E J
E G
V Z
V R
Z W
E S
D Z
S W
R J
S Z
R G
G P
D W
E P
S J
Z J
E V
V P
D P
D S
P W
V D
D G
G S
V G
E R
R S
P Z
V W
E W
V S
R W
D R
G Z
R Z
E Z
P J
6 11
N A U Q B Y 
A B
N A
A Y
U B
A Q
B Y
N Y
U Y
Q Y
N Q
N B
10 25
O S D Q N T X U J I 
O N
S I
D T
O U
T X
U J
S J
S X
D X
O X
N J
X U
O Q
X I
S U
N X
S Q
Q X
D Q
Q N
D U
Q T
D J
X J
S N
6 13
H Y K J W V 
H Y
H V
H J
K V
J W
H K
J V
H W
Y V
Y W
K J
Y J
Y K
6 8
H U W I B M 
H B
I M
U B
W B
H W
U I
W I
B M
10 26
C M A E P K O G Z X 
P O
C M
E O
M A
M O
E G
A O
C G
P K
K O
A E
G Z
C O
Z X
E K
K G
C Z
O X
P G
A P
E Z
M K
C A
C P
M Z
E X
7 21
K W Z C P A N 
Z P
C P
W N
Z A
P A
K C
P N
C N
K W
Z C
W A
W P
K Z
K A
A N
C A
K P
K N
W Z
W C
Z N
10 34
L O U Q H S G Y R E 
H E
L U
S R
Q H
O R
G R
Q S
O G
Y R
U G
L R
O S
L H
H Y
Q E
O H
S G
H S
H G
U Q
U S
Q Y
U Y
L Q
U E
S E
Y E
L G
H R
G E
G Y
R E
O E
U R
6 12
Y A C R F U 
R F
C R
Y A
Y U
Y F
F U
Y C
Y R
A U
C U
A F
C F
6 11
M A G P K F 
M P
K F
A G
P F
M F
M K
A F
G K
P K
M G
A P
5 7
A M W D I 
A D
A I
M W
W D
A W
A M
W I
5 7
K E I V B 
E I
I B
V B
K B
E V
K E
E B
4 4
Z J N H 
J N
Z N
N H
Z H
7 17
F D T R G M O 
F O
R M
T R
D R
G O
T M
R O
T O
R G
D G
T G
F M
G M
D O
F D
M O
F T
7 11
R G A D O F T 
O T
G O
D O
R O
D F
G T
R T
R D
G A
A D
F T
10 32
L E J Q B I W U V G 
J Q
L E
E Q
J G
L Q
J B
E J
W V
E B
L W
I W
B V
Q V
W G
J W
V G
B W
W U
E I
L G
L I
B G
E V
B I
L B
E G
Q U
Q I
L U
J U
I G
E W
4 4
D M S U 
M S
M U
D M
D U
8 18
O M V H T X G L 
V H
M H
H T
M T
H L
T G
V L
O X
O M
M G
H G
V T
T L
O L
O V
M X
V G
M V
9 33
Q G N L I E S K F 
N E
Q N
N I
G K
E F
G S
E S
K F
N F
N K
N S
G I
Q F
I S
Q S
I F
L S
L I
G F
Q G
L E
L F
S K
S F
L K
G L
N L
Q L
Q I
I K
Q E
I E
E K
8 27
R N G J F O E T 
N E
R T
O T
N G
G E
N F
J F
N T
F O
O E
G J
R O
F E
G O
E T
N O
R N
G T
J O
R J
J T
R E
N J
F T
R F
R G
J E
8 15
B Z S Q A R O X 
R X
S O
Q O
Q A
B S
B R
B Z
A O
O X
S A
Q R
S Q
B A
B O
A X
4 4
B A X T 
A T
B A
B T
X T
4 5
L D U B 
D U
L U
D B
U B
L B
8 25
C T V P G Q A D 
T P
P D
T V
C Q
G D
T A
G Q
C D
Q A
T Q
V Q
V G
P A
C G
G A
C P
P Q
C T
P G
V P
T G
V A
C A
C V
Q D
4 5
J U A K 
J K
J A
U A
A K
J U
8 26
U H L K E R V Z 
K E
R V
E Z
U Z
U K
K V
U H
V Z
R Z
H V
E V
K R
H R
U E
L Z
E R
H K
U V
K Z
L E
L R
H E
U R
H L
L V
H Z
7 15
X P L V W T I 
L V
P L
V I
X V
P W
L W
W T
X I
P I
W I
X L
V T
V W
X P
X T
10 25
T P J N Z W E K F H 
P W
P H
K H
N Z
J N
T J
E K
T H
F H
J K
J F
W K
Z W
N H
W F
P N
P F
T F
N F
W E
W H
N K
K F
N W
T N
6 8
O F I W M N 
F N
O M
O W
I M
F I
F M
W M
M N
5 10
S D F E X 
D F
S F
S X
D X
D E
E X
S D
F E
S E
F X
