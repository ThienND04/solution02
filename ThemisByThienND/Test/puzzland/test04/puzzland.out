KQYP
UERI
EGDPJRSVZW
NQABUY
ONJDQTXUSI
HJWYKV
HWBUIM
CAEGKPOMZX
KACPWZN
LGHORSQUYE
YAFRCU
MGAPKF
IMPOSSIBLE
IMPOSSIBLE
IMPOSSIBLE
FDGMRTO
ROGADFT
LBEIQJUWVG
IMPOSSIBLE
OXMGHTVL
QEIGKLNSF
REFJGNOT
IMPOSSIBLE
IMPOSSIBLE
LUDB
CAGPTVQD
JUAK
UEHKRLVZ
XLPWTVI
TFJKEWZNPH
OWMIFN
SDEFX
